### PR TITLE
Add task that will dump buckets that are funded, archived and not paid

### PR DESCRIPTION
**Pull request by chime-mu**
@jlopker Will you review this?

The point is to be able to dump a CSV file that gives us information on all buckets that will change once we deploy https://github.com/cobudget/cobudget-ui/pull/365

### CODE REVIEW

**Comment by chime-mu** [comment]
@jlopker You ok with me merging this to master and deploy to prod?

File: lib/tasks/csv.rake [hunk 1]
```diff
@@ -14,4 +14,22 @@ namespace :csv do
     end
   end
 
+  desc "export all buckets that is funded and has archived_at set as CSV"
```
Comment (jlopker) [inline lib/tasks/csv.rake]: "export all buckets that are funded and have archived_at set to a CSV"

**Comment by jlopker** [comment]
@chime-mu sounds good to me! just one small comment about the desc. what will we do once we have this csv?

**Comment by chime-mu** [comment]
@jlopker I imagine it might be handy for communication with the groups that might experience changes when we deploy the waiting frontend fixes.

File: lib/tasks/csv.rake (hunk 1 above)
Comment (chime-mu) [reply lib/tasks/csv.rake]: @jlopker Thanks 👍

**Pull request merged**